alpine-nix-rails-nginx/example-rails-template.rb

```diff
@@ -26,16 +26,42 @@
 web: bundle exec puma --quiet --config config/puma.rb
 PROCFILE
 
+file 'config/postgres.sh', <<-'POSTGRES'
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
```

Comment:
I would be ok with this, but I think we can do it in a separate PR. I will make an issue for it.